nautobot/extras/api/views.py

```diff
@@ -435,169 +426,23 @@ def _create_schedule(serializer, data, job_model, request, celery_kwargs=dict, t
     # scheduled for.
     scheduled_job = ScheduledJob(
         name=name,
-        task="nautobot.extras.jobs.scheduled_job_handler",
+        task=job_model.job_class.registered_name,
         job_class=job_model.class_path,
         job_model=job_model,
         start_time=time,
-        description=f"Nautobot job {name} scheduled by {request.user} on {time}",
-        kwargs=task_kwargs,
+        description=f"Nautobot job {name} scheduled by {user} for {time}",
+        kwargs=data,
         interval=type_,
         one_off=(type_ == JobExecutionType.TYPE_FUTURE),
-        user=request.user,
-        approval_required=job_model.approval_required,
+        user=user,
+        approval_required=approval_required,
         crontab=crontab,
         queue=task_queue,
     )
     scheduled_job.save()
     return scheduled_job
 
 
-def _run_job(request, job_model):
```

Comment:
This doesn't need to be a separate method anymore since the legacy job view was removed. Moved this over to the `JobViewSet.run` method